packages/api/src/platforms/vtex/utils/skuVariants.ts

```diff
@@ -12,10 +12,12 @@ type FormattedSkuVariant = {
   value: string
 }
 
+const SKU_IMAGE_LABEL = 'skuvariation'
```

Comment:
I just created a const out of that magic value of `skuvariation`. I did not know that we had an specific label for SKU Selector images.